certificate-authority/store/mongodb/signingRecords.go

```diff
@@ -113,7 +116,7 @@ func (s *Store) DeleteSigningRecords(ctx context.Context, owner string, query *s
 	}
 	res, err := s.Collection(signingRecordsCol).DeleteOne(ctx, toSigningRecordsQueryFilter(owner, &q))
 	if err != nil {
-		return -1, fmt.Errorf("cannot remove signing record: %w", err)
+		return -1, pkgErrors.NewError(ErrCannotRemoveSigningRecord, err)
```

Comment:
why you don't use multierror ?

Reply:
I guess that might work too. I was thinking of `multierror` as multiple errors from the same layer, whilst I just wanted to wrap an existing error. So in my head with multierror I can have multiple unrelated errors whilst this should be a single wrapped error, so when some upper layer wraps it further and then checks it by error.Is(ErrCannotRemoveSigningRecord, err) it should return true. However, that will also holds when `multierror` is used. So it seems that the only difference is the formatted output - "%s: %s" vs "\n\t%s\n\t%s" used by `multierror`, which doesn't warrant a new package.